1-js/01-getting-started/2-manuals-specifications/article.md

```diff
@@ -1,42 +1,42 @@
 
-Also, if you're in developing for the browser, then there are other specs covered in the [second part](info:browser-environment) of the tutorial.
+इसके अलावा, यदि आप ब्राउज़र के लिए विकसित कर रहे हैं, तो ट्यूटोरियल के [दूसरे भाग] (जानकारी: ब्राउज़र-पर्यावरण) में शामिल अन्य चश्मा हैं।
 
-## Manuals
+## मैनुअल
 
-- **MDN (Mozilla) JavaScript Reference** is a manual with examples and other information. It's great to get in-depth information about individual language functions, methods etc.
+- **एमडीएन (मोज़िला) जावास्क्रिप्ट संदर्भ** उदाहरण और अन्य जानकारी के साथ एक मैनुअल है। व्यक्तिगत भाषा के कार्यों, विधियों आदि के बारे में गहराई से जानकारी प्राप्त करना बहुत अच्छा है।
 
-    One can find it at <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference>.
+    कोई इसे <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference> पर पा सकता है।
 
-    Although, it's often best to use an internet search instead. Just use "MDN [term]" in the query, e.g. <https://google.com/search?q=MDN+parseInt> to search for `parseInt` function.
+    हालांकि, इसके बजाय अक्सर इंटरनेट खोज का उपयोग करना सबसे अच्छा है। बस क्वेरी में "एमडीएन [टर्म]" का उपयोग करें, उदा। {`parseInt`} फ़ंक्शन की खोज करने के लिए <https://google.com/search?q=MDN parseInt>।
 
 
-- **MSDN** – Microsoft manual with a lot of information, including JavaScript (often referred to as JScript). If one needs something specific to Internet Explorer, better go there: <http://msdn.microsoft.com/>.
+- **MSDN** – Microsoft बहुत सारी जानकारी के साथ मैनुअल करता है, जिसमें जावास्क्रिप्ट (अक्सर JScript के रूप में संदर्भित) शामिल है। अगर किसी को इंटरनेट एक्सप्लोरर के लिए कुछ विशिष्ट चाहिए, तो बेहतर तरीके से वहां जाएं: <http://msdn.microsoft.com/>।
 
-    Also, we can use an internet search with phrases such as "RegExp MSDN" or "RegExp MSDN jscript".
+    इसके अलावा, हम "RegExp MSDN" या "RegExp MSDN jscript" जैसे वाक्यांशों के साथ एक इंटरनेट खोज का उपयोग कर सकते हैं।
 
-## Compatibility tables
+## संगतता टेबल
 
-JavaScript is a developing language, new features get added regularly.
+जावास्क्रिप्ट एक विकासशील भाषा है, नई सुविधाओं को नियमित रूप से जोड़ा जाता है।
 
-To see their support among browser-based and other engines, see:
+ब्राउज़र-आधारित और अन्य इंजनों के बीच उनका समर्थन देखने के लिए, देखें:
 
-- <http://caniuse.com> - per-feature tables of support, e.g. to see which engines support modern cryptography functions: <http://caniuse.com/#feat=cryptography>.
-- <https://kangax.github.io/compat-table> - a table with language features and engines that support those or don't support.
+- <http://caniuse.com> - समर्थन की प्रति-सुविधा तालिका, उदा। यह देखने के लिए कि कौन से इंजन आधुनिक क्रिप्टोग्राफी कार्यों का समर्थन करते हैं: <http://caniuse.com/
+- <https://kangax.github.io/compat-table> - भाषा सुविधाओं और इंजनों के साथ एक तालिका जो समर्थन करती है या समर्थन नहीं करती है।
```

Comment:
Let's use more noun in the sentence. Pronounce are somewhat confusing here.

Reply:
was not able to rectify this one.